homeassistant/components/homematicip_cloud/device.py

```diff
@@ -100,4 +114,14 @@ def device_state_attributes(self):
             attr[ATTR_RSSI_DEVICE] = self._device.rssiDeviceValue
         if hasattr(self._device, "rssiPeerValue") and self._device.rssiPeerValue:
             attr[ATTR_RSSI_PEER] = self._device.rssiPeerValue
+        if hasattr(self._device, "deviceOverheated") and self._device.deviceOverheated:
```

Comment:
See above. Make another dict that maps device attributes to state attribute keys.

Reply:
Done